### ISSUE TITLE
Offsite Backups - Error due to Size

### ISSUE THREAD

**Issue by artfcl-intlgnce** [artfcl-intlgnce/XXMC#126]
AutoSaveWorld plugin is bugging out about the size of the backup when we dont enable compression. 
When enabling compression, the entire server lags. 
When disabling compression on the backup, we get an error that there are too many files (even after adding exluded folders to the exclusions list including the DynMap tile folder)

previously time was 1 backup an hour. if we make 1 every 12 hours or so, with a transfer limit of 500kbps (1gb/30mins roughly), we may not have those issues.
I need to look into this further, may need assistance from Scarsz to see how we can best have backups push to our Premium Dropbox

**Comment by Scarsz** [comment]
Why Dropbox over S3

**Comment by artfcl-intlgnce** [comment]
Because there are no open src libraries for SFTP that devs are using for their backup plugins, all that was available was straight FTP (except 1). We tried S3 remember, the only plugin that had SFTP compatibility was AutoSaveWorld, and SFTP wasn't working (even contacting the dev he blamed it on the lack of libraries).  That is why we pivoted to Dropbox. 

We are now moving off AutoSaveWorld (which unless we're developing a custom backup plugin, we'd be in the same predicament now).

Anyway, now we're using MineBackup + Local + Dropbox instead of AutoSaveWorld + Local + Dropbox and its working great
